README.md

```diff
@@ -44,7 +44,12 @@ The conda dependencies are installed in smaller conda environments automatically
 git clone https://github.com/transXpress/transXpress.git
 ~~~~
 
-2. Install [Miniconda3](https://conda.io/en/latest/miniconda.html)
+2. Install [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge)
+~~~~
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+bash Mambaforge-$(uname)-$(uname -m).sh
```

Comment:
This $(uname) trick doesn't work on macOS. Recommend making it a hardcoded shell variable, to the linux x64 one.

Reply:
Interesting, the [miniforge documentation](https://github.com/conda-forge/miniforge#unix-like-platforms-mac-os--linux) says it should work 🤔If it really is the case that it does not work I can definetelly change the downloaded filename to some arbitrary thing like `latest` instead of `$(uname)`.